samples/net/sockets/big_http_download/src/ca_certificate.h

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CA_CERTIFICATE_H__
+#define __CA_CERTIFICATE_H__
+
+#define CA_CERTIFICATE_TAG 1
+
+/* By default only certificates in DER format are supported. If you want to use
+ * certificate in PEM format, you can enable support for it in Kconfig.
+ */
+
+/* Let�s Encrypt Authority X3 */
+static const unsigned char ca_certificate[] = {
```

Comment:
Same comment as I had with echo-client/server sample, please place the cert into a file and just include it here. There is the generate_inc_file_for_target() CMake macro for this.